ui/app/mainui/SendModalHandler.qml

```diff
@@ -545,6 +549,34 @@ QtObject {
                     return ""
                 }
 
+                property int indexOfTxPathUnderReview: -1
+                readonly property bool reviewingLastTxPath: !!handler.fetchedPathModel && handler.indexOfTxPathUnderReview === handler.fetchedPathModel.rowCount() - 1
+                readonly property var txPathUnderReview: {
+                    if (!handler.fetchedPathModel || handler.indexOfTxPathUnderReview < 0 || handler.indexOfTxPathUnderReview >= handler.fetchedPathModel.rowCount()) {
+                        return
+                    }
+                    return SQUtils.ModelUtils.get(handler.fetchedPathModel, handler.indexOfTxPathUnderReview)
+                }
+
+                property bool approvalForTxPathUnderReviewReviewed: false
+                readonly property bool reviewApprovalForTxPathUnderReview: !!handler.txPathUnderReview
+                                                                           && handler.txPathUnderReview.approvalRequired
+                                                                           && !handler.approvalForTxPathUnderReviewReviewed
+
+                readonly property var filteredFlatNetworksModel: SortFilterProxyModel {
```

Comment:
this already exists as root.filteredFlatNetworksModel

Reply:
That's the rebase thing, I didn't add this code, will remove it now.